examples/dev-example/src/browser/api-test-menu.ts

```diff
@@ -781,74 +810,74 @@ export class ApiTestMenuContribution implements MenuContribution, CommandContrib
             }
         });
         commands.registerCommand(CloseCoffeeEcoreCommand, {
             execute: () => {
                 this.modelServerClient
                     .close('Coffee.ecore')
-                    .then((response: any) => this.messageService.info(printResponse(response)));
+                    .then(result => this.printResponse('close', result));
             }
         });
         commands.registerCommand(ValidationCoffeeEcoreCommand, {
             execute: () => {
                 this.modelServerClient
-                    .validation('Coffee.ecore')
-                    .then((response: any) => this.messageService.info(printResponse(response)));
+                    .validate('Coffee.ecore')
+                    .then(result => this.printResponse('validate', result));
             }
         });
         commands.registerCommand(ValidationMarkersCoffeeEcoreCommand, {
             execute: () => {
                 this.modelServerClient
-                    .validation('Coffee.ecore')
-                    .then((response: any) => {
-                        const message = createMarkersFromResponseBody(this.diagnosticManager, new URI('Coffee.ecore'), response);
+                    .validate('Coffee.ecore')
+                    .then(result => {
+                        const message = createMarkersFromDiagnostic(this.diagnosticManager, new URI('Coffee.ecore'), result);
                         this.messageService.info(message);
                     });
             }
         });
         commands.registerCommand(ValidationConstraintsCoffeeEcoreCommand, {
             execute: () => {
                 this.modelServerClient
-                    .validationConstraints('Coffee.ecore')
-                    .then((response: any) => this.messageService.info(printResponse(response)));
+                    .getValidationConstraints('Coffee.ecore')
+                    .then(result => this.printResponse('getValidationConstraints', result));
             }
         });
         commands.registerCommand(SubscribeWithValidationCoffeeEcoreCommand, {
             execute: () => {
-                this.modelServerClient.subscribeWithValidation('Coffee.ecore');
+                this.modelServerClient.subscribe('Coffee.ecore', { livevalidation: true });
             }
         });
         commands.registerCommand(SubscribeAndKeepAliveCommand, {
             execute: () => {
-                this.modelServerClient.subscribeWithTimeout('Coffee.ecore', 60000);
+                this.modelServerClient.subscribe('Coffee.ecore', { timeout: 60000 });
             }
         });
         commands.registerCommand(SubscribeAndKeepAliveXmiCommand, {
             execute: () => {
-                this.modelServerClient.subscribeWithTimeoutAndFormat('Coffee.ecore', 60000, 'xmi');
+                this.modelServerClient.subscribe('Coffee.ecore', { livevalidation: true, timeout: 60000, format: 'xmi' });
```

Comment:
Subscription in `xmi` format basically works, but incremental updates are not correctly received, I get the following error in the log:
`root ERROR Notification handler 'onMessage' failed with message: Cannot map to given type. Given parameter failed the type guard check!`

Reply:
Thanks, this is resolved now.